src/semeio/forward_models/design_kw/design_kw.py

```diff
@@ -178,3 +184,41 @@ def rm_genkw_prefix(paramsdict, ignoreprefixes="LOG10_"):
         for keyvalue in keyvalues
         if keys.count(keyvalue[0]) == 1
     }
+
+
+def validate_configuration(
+    template_file_name: str, parameters_file_name: str = "parameters.txt"
+) -> None:
+    try:
+        if not os.path.exists(template_file_name):
+            raise ValueError(
+                f"Argument template_file: '{template_file_name}' is not an existing file"
+            )
+
+        if not os.path.exists(parameters_file_name):
+            raise ValueError(
+                f"Could not find required parameters file from DESIGN2PARAMS: '{parameters_file_name}'"
+            )
+        with open(parameters_file_name, encoding="utf-8") as parameters_file:
```

Comment:
This can fail if you don't have read permissions for the parameters_file_name and crash ert instead of showing a warning in the GUI